index.html

```diff
@@ -151,6 +151,10 @@ <h2>The Club</h2>
           <td><a href="https://codeberg.org/kiesel-js/kiesel">Kiesel</a></td>
           <td>a JavaScript engine in Zig, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://github.com/so-dang-cool/dt">dt</a></td>
+          <td>an AWK-like utility but with semantics like Forth, Factor, or Joy. Also features built-in quotes from Red Green. Made for fun!</td>
```

Comment:
Sorry this is too long. How about:

```suggestion
          <td>a concatenative AWK-like language, for fun!</td>
```